### PR TITLE
added functionality for SetCapsLockState

### PR DESCRIPTION
**Pull request by Glutenberg**
CapsLock cannot be reliably toggled with Key_Press(). AHK has a method called SetCapsLockState. 

I have added code to allow for this method to be used.

### CODE REVIEW

**Comment by spyoungtech** [comment]
Hi @Glutenberg #101 was merged in. Can you uncomment the test case now?

File: tests/unittests/test_keyboard.py [hunk 1]
```diff
@@ -21,71 +23,76 @@ def setUp(self):
...
+        """
         self.notepad.activate()
-        self.ahk.send_raw('=foo')
-        assert b'=foo' in self.notepad.text
+        self.ahk.send_raw("=foo")
+        assert b"=foo" in self.notepad.text
+
+    # def test_set_capslock_state(self):
+    #     self.ahk.set_capslock_state("on")
+    #     assert self.ahk.key_state("CapsLock", "T")
```
Comment (spyoungtech) [inline tests/unittests/test_keyboard.py:77]: Let's uncomment this now that `key_state` should work properly :-)

**Review by spyoungtech — CHANGES_REQUESTED** [review]
Just need to see the passing test and I think we can merge this in.

**Comment by Glutenberg** [comment]
Should be good now

**Review by spyoungtech — APPROVED** [review]

**Comment by spyoungtech** [comment]
Awesome! Thanks for your contribution @Glutenberg

**Pull request merged**